### PR TITLE
finished seeding/routing database

### PR DESCRIPTION
**Pull request by RaymondLi-1993**
PR request. routes on backend with seed data

### CODE REVIEW

File: app/controllers/drinks_controller.rb [hunk 1]
```diff
@@ -9,9 +9,23 @@ def index
...
     end
 
+    
 
+    private
+    def drinks_params
+        params.require(:drink).permit(:name, :img_url, :ingredients, :steps, :api_id)
+    end 
 
+    def favorite_drinks_params
```
Comment (minachuong) [inline app/controllers/drinks_controller.rb]: is this being used?

File: app/controllers/drinks_controller.rb [hunk 2]
```diff
@@ -9,9 +9,23 @@ def index
     end
 
     def create
-        Drinks.create(name:params[:name])
+        #current_user to use when not in testing anymore
+        drink = Drink.find_by(api_id:params[:api_id])
+        drink = Drink.create(drinks_params) if drink.nil? 
+        favorite = FavoriteDrink.create(user: params[:user_id], drink: drink)
```
Comment (minachuong) [inline app/controllers/drinks_controller.rb:15]:
```suggestion
        favorite = FavoriteDrink.create(user: current_user.id, drink: drink) 
```
The devise already allows you to access the current_user so it doesn't need to be provided by params

**Review by minachuong — APPROVED** [review]
please see comments

File: app/controllers/drinks_controller.rb (hunk 2 above)
Comment (RaymondLi-1993) [reply app/controllers/drinks_controller.rb:15]: Hey, Mina I was just doing this for testing, when we actually hook the front-end to backend ill switch it over to current_user.

File: app/controllers/drinks_controller.rb (hunk 1 above)
Comment (RaymondLi-1993) [reply app/controllers/drinks_controller.rb]: It is not. I will delete it right now.

**Pull request merged**